README.md

```diff
@@ -28,11 +31,15 @@ officially supported languages have well-supported Avro libraries:
 |Python|[Apache Avro Python](https://avro.apache.org/docs/current/gettingstartedpython.html)|
 |Ruby|[AvroTurf](https://github.com/dasch/avro_turf)|
 
-## Resources
-
-**Documentation**
+## Documentation and Blog Posts
+- [Pub/Sub API Developer Guide](https://developer.salesforce.com/docs/platform/pub-sub-api/overview)
+- [Salesforce Architects Blog Post](https://medium.com/salesforce-architects/announcing-pub-sub-api-generally-available-3980c9eaf0b7)
 - [Salesforce Developers Blog Post](https://developer.salesforce.com/blogs/2021/07/pub-sub-api-building-event-driven-integrations-just-got-even-easier)
 - [Salesforce Developers Podcast Episode](https://developer.salesforce.com/podcast/2021/08/episode-93-pub-sub-api-with-emmett-chen-ran)
```

Comment:
Not directly related to this PR, but this link doesn't seem to work for me anymore, I'm just getting a message that says `Get notified of new episodes with the new Salesforce Developers Slack app.`. Would we want to either fix or remove this link as part of this PR?

Reply:
Remove last two links